site/content/how-to/data-plane-configuration.md

```diff
@@ -157,3 +157,43 @@ as arguments and add `/bin/sh` as the command. The deployment manifest should lo
   - /bin/sh
 ...
 ```
+
+### Configure Rewrite Client IP Settings
```

Comment:
```suggestion
## Configure Rewrite Client IP Settings
```
Otherwise this gets nested under the log level section.

I'm also wondering how we can rephrase this title to include PROXY protocol, since that's the main use case.

Maybe "Configuring PROXY protocol and Rewriting Client IP"

Reply:
I agree, I am feel iffy about just mentioning one mode to do so but since it is more widely used. It does make sense. I'll go ahead with your suggestion.